packages/jest-mock/src/index.ts

```diff
@@ -902,4 +1092,5 @@ class ModuleMockerClass {
 }
 
 export type ModuleMocker = ModuleMockerClass;
-module.exports = new ModuleMockerClass(global);
+
+export default new ModuleMockerClass(global);
```

Comment:
this breaks `const jestMock = require('jest-mock');`

```suggestion
export = new ModuleMockerClass(global);
```

You might need to remove the type export at that point, but we can do `typeof` when importing it elsewhere

Reply:
>An export assignment cannot be used in a module with other exported elements.ts(2309)

when trying `export = new ModuleMockerClass(global);` 🤔

Reply:
yeah, have to kill the type exports up until we complete the TS migration